File: EscapeFromTrinityEngineStats/Models/Relationships/ShopPurchasedPassive.cs
using EscapeFromTrinityEngineStats.Models.Instances;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EscapeFromTrinityEngineStats.Models.Relationships
{
    public class ShopPurchasedPassive
    {
        public int ShopRecordId { get; set; }
        public int PassiveInstanceId { get; set; }
        public virtual PassiveInstance PassiveInstance { get; set; }
    }
}

File: EFTEDashboard/src/app/interfaces/relationships/shop-purchased-passive.ts
import { PassiveInstance } from '../Instances/passive-instance';
import { ShopRecord } from '../shop-record';

export interface ShopPurchasedPassive {
	shopRecordId?: number;
	shopRecord?: ShopRecord;
	passiveInstanceId?: number;
	passiveInstance?: PassiveInstance;
}
